pkg/localstore/gc_test.go

```diff
@@ -209,7 +221,7 @@ func TestPinGC(t *testing.T) {
 
 	t.Run("pin Index count", newItemsCountTest(db.pinIndex, pinChunksCount))
 
-	t.Run("gc exclude index count", newItemsCountTest(db.gcExcludeIndex, 0))
+	t.Run("gc exclude index count", newItemsCountTest(db.gcExcludeIndex, 50))
```

Comment:
i will change it to `pinChunksCount`. The problem that it was zero was because that this index would remove the entry in the beginning of gc round in `removeChunksInExcludeIndexFromGC`, but the item would be removed from the index only if it was in the GC index before. Because I changed the way that the test is created, the item doesn't end up in the gc index initially, so it also stays in this index.

Reply:
this whole thing is a bit arcane but we will pull it out when storage incentives are merged